src/core/Global.ttslua

```diff
@@ -650,14 +650,34 @@ end
 ---------------------------------------------------------
 
 function onClick_refreshList()
-  local request = WebRequest.get(SOURCE_REPO .. '/library.json', completed_list_update)
+  local request = WebRequest.get(SOURCE_REPO .. '/content/library.json', completed_list_update)
   requestObj = request
   startLuaCoroutine(Global, 'downloadCoroutine')
 end
 
-function onClick_select(player, params)
-  params = JSON.decode(urldecode(params))
-  local url = SOURCE_REPO .. '/' .. params.url
+function onClick_select(_, params)
+  if type(params) == "string" then
+    params = JSON.decode(urldecode(params))
+  end
+
+  local url = SOURCE_REPO .. '/content/'
+  if params.url ~= nil and params.url ~= "" then
+    -- TODO: change local folder based on selection in option panel
+    -- TODO: better detection of fancreations
+    if string.match(params.url, "............") == "fancreations" then
```

Comment:
This change should be an update in the library.json file, not done in code.  Especially with version mismatch concerns it will be very hard to change.